src/mscorlib/corefx/System/Globalization/CompareInfo.cs

```diff
@@ -37,6 +37,10 @@ public enum CompareOptions
         OrdinalIgnoreCase = 0x10000000,   // This flag can not be used with other flags.
         StringSort = 0x20000000,   // use string sort method
         Ordinal = 0x40000000,   // This flag can not be used with other flags.
+        
+        // UpperCaseFirst = 0x01000000
```

Comment:
Isn't the right thing here to expose it as a real API, for both Windows and Unix? We can streamline the process and do it via email if necessary.